programming/editors/vscode.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: page
+title: VSCode
+---
+
+# Visual Studio Code
+
+Visual Studio Code (known commonly as VSCode) is a free code editor developed by
+Microsoft. As such it has [extensive documentation][vscode-docs] about its
+general use, which we will not repeat here. Instead this page will focus on the
+ways to get the most out of VSCode when developing code for Student Robotics.
+
+[vscode-docs]: https://code.visualstudio.com/docs
+
+## Python Extension
+
+While VSCode has some built-in support for developing Python, we recommend that
+you additionally install the official [Python extension][ms-python.python]. This
+brings more comprehensive support and enables everything else we will document
+here.
+
+[ms-python.python]: https://marketplace.visualstudio.com/items?itemName=ms-python.python
+
+## Code Completions
+
+In order for VSCode to pick up the `sr.robot` library and offer completions
```

Comment:
Do you think it's worth linking to the wiki page for code completions here (and syntax highlighting above)? Just in case the user doesn't read the index page.

Reply:
Hrm, perhaps.

Reply:
bef42d7